devicemodel/bios/VSBL.txt

```diff
@@ -0,0 +1,7 @@
+Version 0.8
```

Comment:
How about we split the content of this file into two separate files; one called `changelog` and another one called `MD5SUM`? That would be a little more self-explanatory than this file today and the `MD5SUM` could also be used directly (`md5sum -c MD5SUM`)